critiquebrainz/frontend/views/profile_apps.py

```diff
@@ -37,13 +38,20 @@ def create():
 @profile_apps_bp.route('/<client_id>/edit', methods=['GET', 'POST'])
 @login_required
 def edit(client_id):
-    application = OAuthClient.query.get_or_404(client_id)
-    if application.user != current_user:
+    try:
+        application = db_oauth_client.get(client_id)
+    except db_exceptions.NoDataFoundException:
+        abort(404)
+    if application["user_id"] != current_user.id:
         raise NotFound()
     form = ApplicationForm()
     if form.validate_on_submit():
-        application.update(name=form.name.data, desc=form.desc.data,
-                           website=form.website.data, redirect_uri=form.redirect_uri.data)
+        db_oauth_client.update(
+            client_id=application["client_id"],
+            name=form.name.data, desc=form.desc.data,
```

Comment:
Don't stack arguments like that. It's easy to miss it if someone is expecting each to be on a new line.